src/flag_gems/__init__.py

```diff
@@ -10,12 +10,13 @@
 __version__ = "2.1"
 device = runtime.device.name
 aten_lib = torch.library.Library("aten", "IMPL")
+registar = Register
+current_work_registar = None
 
 
-def enable(lib=aten_lib, unused=None):
-    if unused is None:
-        unused = []
-    Register(
+def enable(lib=aten_lib, unused=None, registar=registar):
+    global current_work_registar
```

Comment:
why do we need a global variable to keep the register?

Reply:
theoretically there can't be enable() and use_gems() at the same time. we don't have to know where Register is from.